components/Lessons/Lessons.module.css

```diff
@@ -1,3 +1,10 @@
-.main {
+.container {
   background-color: white;
```

Comment:
```suggestion
```
This is also the default color

Reply:
I need this color, because of the background gradient.